yb-voyager/cmd/analyzeSchema.go

```diff
@@ -964,19 +991,17 @@ func analyzeSchemaInternal() utils.Report {
 	sourceObjList = utils.GetSchemaObjectList(msr.SourceDBConf.DBType)
 	initializeSummaryMap()
 	for _, objType := range sourceObjList {
-		var filePath string
-		if objType == "INDEX" {
-			filePath = filepath.Join(schemaDir, "tables", "INDEXES_table.sql")
+		var sqlInfoArr []sqlInfo
+		filePath := utils.GetObjectFilePath(schemaDir, objType)
+		if objType != "INDEX" {
+			sqlInfoArr = createSqlStrInfoArray(filePath, objType)
 		} else {
-			filePath = filepath.Join(schemaDir, strings.ToLower(objType)+"s")
-			filePath = filepath.Join(filePath, strings.ToLower(objType)+".sql")
+			sqlInfoArr = createSqlStrInfoArray(filePath, objType)
+			otherFPaths := utils.GetObjectFilePath(schemaDir, "PARTITION_INDEX")
+			sqlInfoArr = append(sqlInfoArr, createSqlStrInfoArray(otherFPaths, "PARTITION_INDEX")...)
+			otherFPaths = utils.GetObjectFilePath(schemaDir, "FTS_INDEX")
+			sqlInfoArr = append(sqlInfoArr, createSqlStrInfoArray(otherFPaths, "FTS_INDEX")...)
```

Comment:
What is FTS_INDEX?

Reply:
Full Text Search Indexes.

Reply:
If file doesn't exists, `createSqlStrInfoArray()` will return an empty slice.